filebeat/fileset/config.go

```diff
@@ -0,0 +1,17 @@
+package fileset
+
+// ModuleConfig contains the configuration file options for a module
+type ModuleConfig struct {
+	Module   string                    `config:"module"     validate:"required"`
+	Enabled  *bool                     `config:"enabled"`
+	Filesets map[string]*FilesetConfig `config:"filesets"`
```

Comment:
@tsg Looking at https://github.com/elastic/beats/issues/3159

Shouldn't it be:

```
modules:
- name: nginx
  access:
    close_eof: true
```
I removed the prospector path above. So it is actually `map[string]ProspectorConfig`. Each module in the end is 1 prospector. Or are there other configs inside `access` that could occur?